lua/treesj/langs/init.lua

```diff
@@ -8,6 +8,8 @@ local langs = {
   'css',
   'scss',
   'tsx',
+  'jsx',
```

Comment:
Not needed, because treesitter recognize jsx as javascript.

Reply:
Yeah I saw that too and removed that in `fix failing tests`